roles/zuul/meta/main.yml

```diff
@@ -1,2 +1,12 @@
 dependencies:
   - pip
+  - role: logrotate
+    logrotate_configs:
```

Comment:
I don't think one role has access to another role's defaults, but I can test that.